benchmark/input/benchmark_grammar.g

```diff
@@ -0,0 +1,98 @@
+## This is the grammar file for JuCC
```

Comment:
do we need this to duplicate multiple times🤔

Reply:
incase we change main grammar with out updating benchmarks